miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/NotificationDto.java

```diff
@@ -15,6 +15,16 @@
 import uk.ac.bbsrc.tgac.miso.core.data.type.HealthType;
 import uk.ac.bbsrc.tgac.miso.core.data.type.PlatformType;
 
+/**
+ * A "run" as seen by Run Scanner
+ *
+ * Run Scanner collects information about runs and reports them to MISO for incorporation. This class is distinct from {@link RunDto}
+ * because there is not a direct mapping between runs in MISO and runs as they can be detected on disk ("notifications"). First, runs have
+ * added information such as change logs, security profiles, and notes that notifications should never include. Second, the notification is
+ * actually a composite of run, container, and partition information and this has to be expanded when written to MISO in a non-trivial way.
+ * Finally, notifications contain best-effort information gleaned from the sequencer output, but it is not always possible to correctly
+ * detect information; if a human changes certain properties of a run, it should be updated from automation.
```

Comment:
`should be updated from automation` -> `should not be overwritten by automation`?